internal/project/repository/eventsourcing/eventstore.go

```diff
@@ -51,23 +52,31 @@ func (es *ProjectEventstore) CreateProject(ctx context.Context, project *proj_mo
 	return ProjectToModel(repoProject), nil
 }
 
-func (es *ProjectEventstore) UpdateProject(ctx context.Context, existingProject *proj_model.Project, project *proj_model.Project) (*proj_model.Project, error) {
+func (es *ProjectEventstore) UpdateProject(ctx context.Context, project *proj_model.Project) (*proj_model.Project, error) {
 	if !project.IsValid() {
 		return nil, caos_errs.ThrowPreconditionFailed(nil, "EVENT-9dk45", "Name is required")
 	}
+	existingProject, err := es.ProjectByID(ctx, &proj_model.Project{ObjectRoot: models.ObjectRoot{ID: project.ID, Sequence: 0}})
```

Comment:
I would use the `NewProject` function because its easier to read

Reply:
will be solved with next mr